bindings/java/src/org/sleuthkit/datamodel/Blackboard.java

```diff
@@ -696,7 +697,10 @@ private boolean attributesMatch(Collection<BlackboardAttribute> fileAttributesLi
 	 *
 	 * @throws TskCoreException exception thrown if a critical error occurs
 	 *                          within tsk core
+	 * 
+	 * @deprecated Use the newDataArtifact() API to create an artifact with attributes.
 	 */
+	@Deprecated
```

Comment:
Can't we replace the body of this method with a call to the newDataArtifact() API?